src/core/core/service_api_c.cpp

```diff
@@ -128,11 +128,6 @@ DSN_API int dsn_threadpool_code_max()
     return ::dsn::utils::customized_id_mgr<::dsn::threadpool_code2_>::instance().max_value();
 }
 
-DSN_API int dsn_threadpool_get_current_tid() 
```

Comment:
this is used in other language bindings (e.g., Java now), so it is cannot be removed.